singleprocess.go

```diff
@@ -160,7 +160,9 @@ func (e *singleprocessEnv) RegisterComponentToStart(_ context.Context, _ string,
 	return nil
 }
 
-func (e *singleprocessEnv) GetComponentsToStart(context.Context, *call.Version) ([]string, *call.Version, error) {
+func (e *singleprocessEnv) GetComponentsToStart(ctx context.Context, _*call.Version) ([]string, *call.Version, error) {
```

Comment:
Oops, I missed a space in my suggested edit
```suggestion
func (e *singleprocessEnv) GetComponentsToStart(ctx context.Context, _ *call.Version) ([]string, *call.Version, error) {
```